src/jit/compiler.cpp

```diff
@@ -4724,68 +4745,55 @@ void Compiler::compCompile(void** methodCodePtr, ULONG* methodCodeSize, JitFlags
         {
             if (doSsa)
             {
-                fgSsaBuild();
-                EndPhase(PHASE_BUILD_SSA);
+                DoPhase(this, PHASE_BUILD_SSA, &Compiler::fgSsaBuild);
             }
 
             if (doEarlyProp)
             {
-                /* Propagate array length and rewrite getType() method call */
-                optEarlyProp();
-                EndPhase(PHASE_EARLY_PROP);
+                DoPhase(this, PHASE_EARLY_PROP, &Compiler::optEarlyProp);
```

Comment:
In some places you removed comments before calls to DoPhase (e.g., PHASE_EARLY_PROP, PHASE_VALUE_NUMBER, PHASE_VN_COPY_PROP, etc.) and in some places you left them alone (e.g., PHASE_OPTIMIZE_LOOPS, PHASE_CLONE_LOOPS, PHASE_UNROLL_LOOPS, etc.) I think we should be consistent.